运行./build-wasm-simd-kws.sh 编译失败

我在Linux机器上，跟随wasm下的kws示例教程，运行./build-wasm-simd-kws.sh 提示编译失败：
/sherpa-onnx/wasm/kws/sherpa-onnx-wasm-main-kws.cc:19:15: error: static assertion failed due to requirement 'sizeof(SherpaOnnxOnlineModelConfig) == sizeof(SherpaOnnxOnlineTransducerModelConfig) + sizeof(SherpaOnnxOnlineParaformerModelConfig) + sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) + 9 * 4':
   19 | static_assert(sizeof(SherpaOnnxOnlineModelConfig) ==
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   20 |                   sizeof(SherpaOnnxOnlineTransducerModelConfig) +
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   21 |                       sizeof(SherpaOnnxOnlineParaformerModelConfig) +
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   22 |                       sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) + 9 * 4,
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/hupan/sherpa-onnx/wasm/kws/sherpa-onnx-wasm-main-kws.cc:19:51: note: expression evaluates to '64 == 60'
   19 | static_assert(sizeof(SherpaOnnxOnlineModelConfig) ==
      |               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
   20 |                   sizeof(SherpaOnnxOnlineTransducerModelConfig) +
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   21 |                       sizeof(SherpaOnnxOnlineParaformerModelConfig) +
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   22 |                       sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) + 9 * 4,
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

--- a/wasm/kws/sherpa-onnx-wasm-main-kws.cc
+++ b/wasm/kws/sherpa-onnx-wasm-main-kws.cc
@@ -19,7 +19,8 @@ static_assert(sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) == 1 * 4, "");
 static_assert(sizeof(SherpaOnnxOnlineModelConfig) ==
                   sizeof(SherpaOnnxOnlineTransducerModelConfig) +
                       sizeof(SherpaOnnxOnlineParaformerModelConfig) +
-                      sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) + 9 * 4,
+                      sizeof(SherpaOnnxOnlineZipformer2CtcModelConfig) + 9 * 4 +
+                      sizeof(SherpaOnnxOnlineNemoCtcModelConfig),
               "");
 static_assert(sizeof(SherpaOnnxFeatureConfig) == 2 * 4, "");
 static_assert(sizeof(SherpaOnnxKeywordSpotterConfig) ==

